next-tavla/app/(admin)/organizations/[id]/page.tsx

```diff
@@ -50,13 +50,13 @@ async function EditOrganizationPage({ params }: TProps) {
 
     return (
         <main className={classes.root}>
-            <Heading1>{organization.name}</Heading1>
-            <LeadParagraph>
+            <Heading1 margin="top">{organization.name}</Heading1>
+            <LeadParagraph margin="none" className="mb-4">
```

Comment:
kunne vi brukt `margin="bottom"` direkte istedenfor?

Reply:
Vi trenger mer spacing enn det som er default for `margin="bottom"`, og `className` overskriver default margin, så det er det samme om det står `"none"` eller `"bottom"`